src/modules/Dropdown/Dropdown.js

```diff
@@ -795,7 +795,8 @@ export default class Dropdown extends Component {
         filteredOptions = search(filteredOptions, searchQuery)
       } else {
         const re = new RegExp(_.escapeRegExp(searchQuery), 'i')
-        filteredOptions = _.filter(filteredOptions, opt => re.test(opt.text))
+        // remove diacritics on search
+        filteredOptions = _.filter(filteredOptions, opt => re.test(opt.text ? _.deburr(opt.text): opt.text))
```

Comment:
Lodash gracefully handles `null` and `undefined` values, so you can safely remove the ternary check here.  The method will always return a string:

![image](https://user-images.githubusercontent.com/5067638/29835051-958c8b5a-8ca5-11e7-9436-0a7ee2e2186a.png)